docs/releases/1.10.3.txt

```diff
@@ -23,3 +23,6 @@ Bugfixes
 
 * Fixed ``QuerySet.bulk_create()`` on PostgreSQL when the number of objects is
   a multiple plus one of ``batch_size`` (:ticket:`27385`).
+
+* Fixed redirecting issue when session destroyed during request processing
```

Comment:
The session state race condition that has occurred here was probable not caused by the user and it's not very suspicious so the docs of SuspiciousOperation does not fit well: "exception is raised when a user has performed an operation that should be considered suspicious from a security perspective". I don't have a better suggestion though.